clio-server/config/docker-compose.yml.ctmpl

```diff
@@ -18,7 +18,7 @@ services:
 
   proxy:
     image: broadinstitute/openidc-proxy:latest
-    hostname: {{env "CLIO_HOST"}}
+    hostname: {{env "CLIO_DOMAIN_NAME"}}
```

Comment:
This is a bit confusing to me as I think about host and domain names as the same things. It looks like in this case `CLIO_DOMAIN_NAME` refers to the fully qualified domain name. Not sure it's worth calling this `CLIO_FQDN` or something similar, but the fact that "domain name" and "host name" are synonyms makes it a little harder to understand.

Reply:
FWIW: I like FQDN too.